.env.development

```diff
@@ -8,3 +8,4 @@ REACT_APP_USE_DOPPLER_LEGACY_FORGOTPASSWORD=false
 REACT_APP_SHOPIFY_URL= https://sfy.fromdoppler.com:4433
 REACT_APP_DOPPLER_API_URL= http://newapiqa.fromdoppler.net
 REACT_APP_REPORTS_URL= https://reportsint.fromdoppler.net:400
+REACT_APP_OFFLINE=false
```

Comment:
Shouldn't "prettier" recognize the space after =?

Reply:
Good catch!